android/iden3coreapi/src/main/java/com/iden3/iden3coreapi/Iden3Identity.kt

```diff
@@ -0,0 +1,155 @@
+package com.iden3.iden3coreapi
+
+import iden3mobile.*
+import java.util.ArrayList
+import java.util.HashMap
+
+class Iden3Identity(private var identity: Identity) {
+
+    /**
+     * Stop an Iden3 identity.
+     *
+     */
+    fun stopIdentity() {
+        identity.stop()
+    }
+
+    /**
+     * Request a claim from an issuer.
+     *
+     * @param issuerUrl: Url of the issuer.
+     * @param data: data needed by the issuer for requesting a claim.
+     * @param ticketListener: Listener of tickets received after requesting the claim.
+     *
+     * @return An Iden3Ticket.
+     */
+    @Throws(Exception::class)
+    fun requestClaim(issuerUrl: String, data: String, ticketListener: Iden3TicketListener?) : Iden3Ticket? {
+        if (ticketListener == null) {
+            return Iden3Ticket(identity.requestClaim(issuerUrl, data))
+        } else {
+            identity.requestClaimWithCb(issuerUrl, data) {
+                ticket, exception ->
+                if (exception != null) {
+                    ticketListener.onError(exception)
+                } else if (ticket != null) {
+                    ticketListener.onTicketReceived(Iden3Ticket(ticket))
+                }
+            }
+        }
+        return null
+    }
+
+    /**
+     * Prove a claim from an verifier.
+     *
+     * @param verifierUrl: url of the verifier.
+     * @param credentialId: credential id needed by the verifier for proving a claim.
```

Comment:
Should be something like: id of the credential that will be used to generate the proof.
The way it works is:
1. The identity (holder) request a claim to an issuer
2. The identity receives and stores the claim (in a DB managed by Go)
3. When an integrator wants to prove a claim/credential, it must pass to Go the ID of the credential (to fetch it from the DB), with this credential, a proof is generated and sended to the verifier

Reply:
done